hw/opentitan/ot_dma.c

```diff
@@ -963,6 +973,25 @@ static void ot_dma_abort(OtDMAState *s)
     timer_mod(s->timer, (int64_t)now);
 }
 
+static void ot_dma_trigger_fatal_error(OtDMAState *s, const char *func,
+                                       int line)
+{
+    const OtDMAOp *op = &s->op;
+
+    error_report("%s: %s: DMA fatal error @ line %d", func, s->ot_id, line);
+    error_report("  %s addr 0x%" DMA_ADDR_FMT " size 0x%" DMA_ADDR_FMT
```

Comment:
I think you still need to drop the `%`s here as they are included in `DMA_ADDR_FMT`


```suggestion
    error_report("  %s addr 0x" DMA_ADDR_FMT " size 0x" DMA_ADDR_FMT
```